scripts/build-zephyr.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+if [ "`whoami`" = "root" ]
+then
+	echo "Running the script as root is not permitted"
+	exit 1
+fi
+
+CALLED=$_
+[[ "${BASH_SOURCE[0]}" != "${0}" ]] && SOURCED=1 || SOURCED=0
+
+SCRIPT_SRC=$(realpath ${BASH_SOURCE[0]})
+SCRIPT_DIR=$(dirname $SCRIPT_SRC)
+TOP_DIR=$(realpath $SCRIPT_DIR/..)
+
+if [ $SOURCED = 1 ]; then
+	echo "You must run this script, rather then try to source it."
+	echo "$SCRIPT_SRC"
+	exit 1
+fi
+
+if [ -z "$HDMI2USB_ENV" ]; then
+	echo "You appear to not be inside the HDMI2USB environment."
+	echo "Please enter environment with:"
+	echo "  source scripts/enter-env.sh"
+	exit 1
+fi
+
+if [ -z "$ZEPHYR_SDK_INSTALL_DIR" ]; then
+	echo "Zephyr SDK not found"
+	echo "Install Zephyr SDK and/or set ZEPHYR_SDK_INSTALL_DIR variable."
+	echo "For details, see: https://www.zephyrproject.org/developers/"
+	exit 1
+fi
+
+# Imports TARGET, PLATFORM, CPU and TARGET_BUILD_DIR from Makefile
+eval $(make env)
+make info
+
+set -x
+set -e
+
+ZEPHYR_REPO=https://github.com/zephyrproject-rtos/zephyr
+ZEPHYR_REPO_BRANCH=master
+
+case $CPU in
+	vexriscv)
+		TARGET_BOARD=arty_litex_vexriscv
+		ZEPHYR_REPO=https://github.com/antmicro/zephyr
+		ZEPHYR_REPO_BRANCH=litex-vexriscv
+		;;
```

Comment:
I think you might want to check `CPU_VARIANT`?

Reply:
There are many acceptable `CPU_VARIANTS`, so the validating `if` is quite long. 
I think, however, that it's better to explicitely list all acceptable options rather than fail on `minimal`/`minimal+debug` variants.

Reply:
I think that makes sense.